packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -139,6 +143,11 @@ const BlockInspectorSingleBlock = ( {
 				</div>
 			) }
 			<InspectorControls.Slot />
+			<InspectorControls.Slot
+				__experimentalGroup="color"
+				label={ __( 'Color' ) }
+				className="color-block-support-panel__inner-wrapper"
```

Comment:
This class doesn't seem to follow our naming guidelines. We should start with the package name followed by the component name and a modified, so something like `block-editor-block-inspector__color-slot`

Reply:
I can create a PR to tidy this up.